tests/e2e/medic-conf-tests.js

```diff
@@ -0,0 +1,54 @@
+const _ = require('lodash');
+const {expect} = require('chai');
+const util = require('util');
+const { API_PORT }=require('../constants');
+const exec = util.promisify(require('child_process').exec);
+
+const actions = ['compile-app-settings','backup-app-settings','convert-app-forms',
+  'convert-collect-forms','convert-contact-forms', 'backup-all-forms','upload-app-forms','upload-collect-forms',
+  'upload-contact-forms', 'upload-resources','upload-branding','upload-partners','upload-custom-translations',
+  'upload-privacy-policies'];
+
+describe('medic-conf supported actions', () => {
+  beforeAll( () =>{
+    // Change the directory
+    try {
+      process.chdir('config/default');
+    } catch (err) {
+      console.error('no need to change directory');
+    }
+  });
+
+  afterAll(() =>{
+    // Change the directory back to cht-core
+    process.chdir('../..');
+  });
+
+  const runCommand = async (action) => {
+    const { stdout, stderr } = await exec(`medic-conf --url=http://admin:pass@localhost:${API_PORT} ${action} --force`);
+    if (stderr) {
+      console.error(`error: ${stderr}`);
+    }
+    return stdout;
+  };
+
+  it('should execute  upload-app-settings', async () => {
+    try {
+      const result = await runCommand('upload-app-settings');
+      expect(result).to.contain(`INFO Settings updated successfully`);
+    } catch (error) {
+      console.log(error.stdout);
+    }
+  });
+	
+  _.forEach(actions,  async (action)  =>{
```

Comment:
You don't need to use _.forEach here, you could just use [Array.prototype.foreach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach). 

As a caveat, using any type of foreach will make your async callbacks run concurrently, instead of serialized. 
If you want to execute the tests, one after the other (I assume you do), you're going to need to use `for .. in ...`. 

Have a look at https://zellwk.com/blog/async-await-in-loops/